docs_user/modules/proc_migrating-mon-from-controller-nodes-verification.adoc

```diff
@@ -9,10 +9,19 @@ After you finish migrating your Ceph Monitor daemons to the target nodes, verify
 * Verify that the {CephCluster} cluster is healthy:
 +
 ----
-[ceph: root@oc0-controller-0 specs]# ceph -s
+$ ceph -s
```

Comment:
@klgill I think this section will be extended when we get feedback from the field.